bt_helper.py

```diff
@@ -80,7 +80,16 @@ def scan(self):
                 arg0 = "org.bluez.Device1",
                 path_keyword = "path")
         for adapter in self._get_objects_by_iface(ADAPTER_IFACE):
-            dbus.Interface(adapter, ADAPTER_IFACE).StartDiscovery()
+            try:
+                dbus.Interface(adapter, ADAPTER_IFACE).StartDiscovery()
+            except Exception as exc:
```

Comment:
I guess the things we should catch here are DBusExceptions, as those are the ones we want to handle. If anything else is raised we should IMHO let it through, as this signifies something weird/bad/important.